pkg/cli/zip_test.go

```diff
@@ -1252,3 +1252,60 @@ func TestCommandFlags(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// This tests the operation of zip over excluded nodes.
+func TestPartialZipForExcludedNodes(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	dir, cleanupFn := testutils.TempDir(t)
+	defer cleanupFn()
+	c := NewCLITest(TestCLIParams{
+		StoreSpecs: []base.StoreSpec{{
+			Path: dir,
+		}},
+	})
+	defer c.Cleanup()
+	zipName := filepath.Join(dir, "debug.zip")
+
+	// We want a low timeout so that the test doesn't take forever;
+	// however low timeouts make race runs flaky with false positives.
+	skip.UnderShort(t)
+	skip.UnderRace(t)
+
+	sc := log.ScopeWithoutShowLogs(t)
+	defer sc.Close(t)
+	// Reduce the number of output log files to just what's expected.
+	defer sc.SetupSingleFileLogging()()
+
+	ctx := context.Background()
+
+	tc := testcluster.StartTestCluster(t, 1, base.TestClusterArgs{
+		ServerArgs: base.TestServerArgs{
+			DefaultTestTenant: base.TestIsSpecificToStorageLayerAndNeedsASystemTenant,
+			Insecure:          true,
+		},
+	})
+	defer tc.Stopper().Stop(ctx)
+
+	_, err := c.RunWithCapture("debug zip --concurrency=1 --exclude-nodes=1 --cpu-profile-duration=0 " + dir + "/debug.zip")
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
NIT

```suggestion
  require.NoError(t, err)
```

(same applies below as well)

Reply:
DONE
<!-- Reviewable comment -OB9hm0E-NW4FVbMSzfb:bb6tyiv -->
<!-- Sent from Reviewable.io -->